sfdx-project.schema.json

```diff
@@ -95,6 +99,9 @@
           "apexTestAccess": {
             "$ref": "#/definitions/packageDirectory.apexTestAccess"
           },
+          "packageMetadataAccess": {
+            "$ref": "#/definitions/packageDirectory.apexTestAccess"
```

Comment:
Should this be .../packageDirectory.packageMetadataAccess ?

Reply:
yes!